docs/administration/Configuration.md

```diff
@@ -42,462 +42,2044 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
+- **Description:** The length of time that the BE waits to exit after the disk hangs.
+
+#### min_cumulative_compaction_failure_interval_sec
+
+- **Default:** 30 seconds
+- **Description:** The minimum time interval at which Cumulative Compaction retries upon failures.
+
+#### size_tiered_level_num
+
+- **Default:** 7 (Number of Levels for Size-tiered Compaction)
+- **Description:** The number of levels for the Size-tiered Compaction strategy. At most one rowset is reserved for each level. Therefore, under a stable condition, there are, at most, as many rowsets as the level number specified in this configuration item.
+
+#### size_tiered_level_multiple
+
+- **Default:** 5 (Multiple of Data Size Between Contiguous Levels in Size-tiered Compaction)
+- **Description:** The multiple of data size between two contiguous levels in the Size-tiered Compaction strategy.
+
+#### size_tiered_min_level_size
+
+- **Default:** 131,072 Bytes
+- **Description:** The data size of the minimum level in the Size-tiered Compaction strategy. Rowsets smaller than this value immediately trigger the data compaction.
+
+#### storage_page_cache_limit
+
+- **Default:** 20%
+- **Description:** The PageCache size. It can be specified as size, for example, `20G`, `20,480M`, `20,971,520K`, or `21,474,836,480B`. It can also be specified as the ratio (percentage) to the memory size, for example, `20%`. It takes effect only when `disable_storage_page_cache` is set to `false`.
+
+#### internal_service_async_thread_num
+
+- **Default:** 10 (Number of Threads)
+- **Description:** The thread pool size allowed on each BE for interacting with Kafka. Currently, the FE responsible for processing Routine Load requests depends on BEs to interact with Kafka, and each BE in StarRocks has its own thread pool for interactions with Kafka. If a large number of Routine Load tasks are distributed to a BE, the BE's thread pool for interactions with Kafka may be too busy to process all tasks in a timely manner. In this situation, you can adjust the value of this parameter to suit your needs.
+
+### Configure BE static parameters
+
+You can only set the static parameters of a BE by changing them in the corresponding configuration file **be.conf**, and restart the BE to allow the changes to take effect.
+
+BE static parameters are as follows.
+
+#### hdfs_client_enable_hedged_read
+
+- **Default**: false
+- **Unit**: N/A
+- **Description**: Specifies whether to enable the hedged read feature. This parameter is supported from v3.0 onwards.
+
+#### hdfs_client_hedged_read_threadpool_size
+
+- **Default**: 128
+- **Unit**: N/A
+- **Description**: Specifies the size of the Hedged Read thread pool on your HDFS client. The thread pool size limits the number of threads to dedicate to the running of hedged reads in your HDFS client. This parameter is supported from v3.0 onwards. It is equivalent to the dfs.client.hedged.read.threadpool.size parameter in the hdfs-site.xml file of your HDFS cluster.
```

Comment:
maybe for parameters that have clear version info, we can add a row called "Supported in xxx" or add the version info to the parameter name, for example, ![image](https://github.com/StarRocks/starrocks/assets/98087056/369dd25d-3381-44ee-83c4-b53a2252c1dd). but I think the current change is great, we can add the version info later

Reply:
yes, Amber also had great suggestions for making this more meaningful. I will copy this and Amber's suggestions into a doc.

cc: @amber-create @evelynzhaojie